### PR TITLE
feat: demo data notebook

### LINKED ISSUE #1351: Notebooks: Demo Data Notebook
As part of our first mile onboarding experience, we allow users to install demo data and see a dashboard. What if we replaced the demo data dashboard with a demo data notebook?

This has also come up in user research. That what people want is an example notebook.

We can serve the notebook some JSON to define the template and make a new path like  /org/:orgid/notebooks/from/:template or something per @drdelambre (Tagging you, Alex, so you can follow this)

### PR DESCRIPTION
**Pull request by mnorriswalker**
For use in our onboarding userpilot

Closes #1351

### CODE REVIEW

File: src/flows/components/FromTemplatePage.tsx [hunk 1]
```diff
@@ -33,6 +33,263 @@ const TEMPLATE_MAP = {
...
+        createdAt: '2021-05-24T22:00:11.407744829Z',
+        id: '031437230cc0b68b',
+        labels: [],
+        links: {
+          labels: '/api/v2/buckets/031437230cc0b68b/labels',
+          members: '/api/v2/buckets/031437230cc0b68b/members',
+          org: '/api/v2/orgs/e859306b862b3e21',
+          owners: '/api/v2/buckets/031437230cc0b68b/owners',
+          self: '/api/v2/buckets/031437230cc0b68b',
+          write: '/api/v2/write?org=e859306b862b3e21&bucket=031437230cc0b68b'
```
Comment (mnorriswalker) [inline src/flows/components/FromTemplatePage.tsx:259]: Not sure how these work with a template... right now these all map to the org that I used to create and export the JSON...
`Member`
`Org`
`Owners`
`Self`
`Write`
?

File: src/flows/components/FromTemplatePage.tsx [hunk 2]
```diff
@@ -33,6 +33,263 @@ const TEMPLATE_MAP = {
...
+        },
+        name: 'Downsampled Website Monitoring',
+        orgID: 'e859306b862b3e21',
```
Comment (mnorriswalker) [inline src/flows/components/FromTemplatePage.tsx:262]: Same here... this is the `OrgID` that I used to create the template...

**Review by mavarius — COMMENTED** [review]
A few things of note. There are some minor linter issues with the indentation and commas. But besides that, I think we may need to inject the appropriate `orgID` and add a step for the user to create the downsampled bucket.

File: src/flows/components/FromTemplatePage.tsx (hunk 1 above)
Comment (mnorriswalker) [reply src/flows/components/FromTemplatePage.tsx:259]: Ok what if we use the `_tasks` bucket? I think it's a system bucket that everyone has

Comment (drdelambre) [reply src/flows/components/FromTemplatePage.tsx:259]: none of these links are needed. feel free to delete the whole object if needed

File: src/flows/components/FromTemplatePage.tsx (hunk 2 above)
Comment (drdelambre) [reply src/flows/components/FromTemplatePage.tsx:262]: is this a different bucket than the existing sample data bucket?

Comment (mnorriswalker) [reply src/flows/components/FromTemplatePage.tsx:262]: Yes, we can switch to `_tasks` bucket which everyone has by default. We can't write to the `Website Monitoring Demo Data Bucket` due to how it was built/implemented.

File: src/flows/components/FromTemplatePage.tsx (hunk 1 above)
Comment (mnorriswalker) [reply src/flows/components/FromTemplatePage.tsx:259]: Done

File: src/flows/components/FromTemplatePage.tsx [hunk 3]
```diff
@@ -33,6 +33,221 @@ const TEMPLATE_MAP = {
...
+              base: '10',
+              bounds: [
+                '',
+                '',
+              ],
+              label: '',
+              prefix: '',
+              scale: 'linear',
+              suffix: '',
+            }
```
Comment (mavarius) [inline src/flows/components/FromTemplatePage.tsx:116]:
```suggestion
            },
```

File: src/flows/components/FromTemplatePage.tsx [hunk 4]
```diff
@@ -33,6 +33,221 @@ const TEMPLATE_MAP = {
...
+                    aggregateFunctionType: 'filter',
+                    key: '_measurement',
+                    values: [],
+                  },
+                ],
+              },
+              editMode: 'builder',
+              name: '',
+              text: '',
+            }
```
Comment (mavarius) [inline src/flows/components/FromTemplatePage.tsx:173]:
```suggestion
            },
```

**Review by mavarius — APPROVED** [review]
Just 2 missed commas but otherwise looks good!

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 56 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1826)

**Comment by genehynson** [comment]
@mnorriswalker @mavarius this PR will need to be rebased/merged with master in order for CI to pass

**Comment by mavarius** [comment]
@genehynson CI is still failing after the rebase.

**Comment by genehynson** [comment]
> @genehynson CI is still failing after the rebase.

yeah Timmy is working on a fix https://influxdata.slack.com/archives/CLHATC50E/p1626129865183700

**Pull request merged**